### PR TITLE
Project plan 4.33

### PR DESCRIPTION
**Pull request by elsazac**
Eclipse SDK project plan 4.33

### CODE REVIEW

File: development/plans/eclipse_project_plan_4_33.xml [hunk 1]
```diff
@@ -0,0 +1,406 @@
...
+			<td rowspan="1">Power 64-bit LE</td>
+			<td rowspan="1">
+				OpenJDK 17.0.8 (LTS)<br/>
+			</td>
+		</tr>
+
+		<!-- ************ Ubuntu ************** -->
+		<tr class="c1">
+			<td rowspan="1">Ubuntu Long Term Support</td>
+		    <td rowspan="1">22.04</td>
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_33.xml:247]: This should be 24.04 - only latest LTS version is supported.

File: development/plans/eclipse_project_plan_4_33.xml [hunk 2]
```diff
@@ -0,0 +1,406 @@
...
+			</td>
+			<td rowspan="1">GTK 3</td>
+		</tr>
+		
+		<!-- ************ Mac ************** -->
+		<tr class="c1">
+			<td rowspan="2">Apple macOS</td>
+			<td rowspan="1">
+				12<br/>
+				13
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_33.xml:264]: MacOS 14 should be added for sure, I would say that only MacOS 12 should be dropped from referenced platforms.

File: development/plans/eclipse_project_plan_4_33.xml [hunk 3]
```diff
@@ -0,0 +1,406 @@
...
+			</td>
+			
+			<td rowspan="1">x86 64-bit</td>
+			
+			<td rowspan="1">
+				OpenJDK 17.0.8 (LTS)<br/>
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_33.xml:270]: Java 17.0.11 is the latest 17 lts version, Please change it everywhere.

File: development/plans/eclipse_project_plan_4_33.xml [hunk 4]
```diff
@@ -0,0 +1,406 @@
...
+				OpenJDK 21 (LTS)<br/>
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_33.xml:271]: Java 21.0.2 is the latest 21 lts version, Please change it everywhere.

File: development/plans/eclipse_project_plan_4_33.xml [hunk 5]
```diff
@@ -0,0 +1,406 @@
...
+
+<p><strong>API Contract Compatibility:</strong> Eclipse SDK 4.33 will be upwards
+  contract-compatible with Eclipse SDK 4.32 except in those areas noted in the
+  <a href="https://help.eclipse.org/2024-09/topic/org.eclipse.platform.doc.isv/porting/eclipse_4_33_porting_guide.html" target="_top">
+    <em>Eclipse 4.33 Plug-in Migration Guide</em>
+  </a>. Programs that use affected APIs and extension points will need to be ported
+  to Eclipse SDK 4.33 APIs. Downward contract compatibility
+  is not supported. There is no guarantee that compliance with Eclipse SDK 4.33
+  APIs would ensure compliance with Eclipse SDK 4.32 APIs. Refer to
+  <a href="https://wiki.eclipse.org/index.php/Evolving_Java-based_APIs">
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_33.xml:336]: https://github.com/eclipse-platform/eclipse.platform/blob/master/docs/Evolving-Java-based-APIs.md is the new home for this document. Please update the link.

**Review by akurtakov — COMMENTED** [review]
Some changes needed.

File: development/plans/eclipse_project_plan_4_33.xml (hunk 1 above)
Comment (elsazac) [reply development/plans/eclipse_project_plan_4_33.xml]: @akurtakov 

Thanks for the review.
Made the changes. Please have a look.

File: development/plans/eclipse_project_plan_4_33.xml [hunk 6]
```diff
@@ -0,0 +1,406 @@
...
+			<td rowspan="2">Apple macOS</td>
+			<td rowspan="1">
+				13<br/>
+				14
+			</td>
+			
+			<td rowspan="1">x86 64-bit</td>
+			
+			<td rowspan="1">
+				Java 17.0.11 (LTS)<br/>
```
Comment (akurtakov) [inline development/plans/eclipse_project_plan_4_33.xml:270]: Please bring back the OpenJDK as it speicifies a concrete distribution, I used Java as as general name for different "distributions" to be updated . 
To state it more clearly it should be the following everywhere:
OpenJDK 17.0.11 (LTS)
OpenJDK 21.0.2 (LTS)
Oracle Java 17.0.11 (LTS)
Oracle Java 21.0.2 (LTS)

Comment (elsazac) [reply development/plans/eclipse_project_plan_4_33.xml]: Done

Comment (elsazac) [reply development/plans/eclipse_project_plan_4_33.xml]: I have addressed the above change as well. So merging this now.

**Pull request merged**